### PR TITLE
Remove the unused 'mavenrepository' command from the maven plugin

### PR DESCRIPTION
**Pull request by eamansour**
## Why?
See [Remove unwanted/unused code #1891](https://github.com/galasa-dev/projectmanagement/issues/1891)

A vulnerability was reported in the `mavenrepository` command's use of SHA-1. On inspection, this command is not used anywhere in the Galasa codebase, so this PR removes it to remediate the vulnerability.

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-maven-plugin/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	<groupId>dev.galasa</groupId>
 	<artifactId>galasa-maven-plugin</artifactId>
 	<packaging>maven-plugin</packaging>
-	<version>0.33.0</version>
```
Comment (techcobweb) [inline galasa-maven-plugin/pom.xml:10]: If we bump that, we need to bump the example output created by `galasactl project create` too.

Comment (techcobweb) [reply galasa-maven-plugin/pom.xml:10]: Does it get pulled-in to the obr or isolated also?

**Review by techcobweb — APPROVED** [review]
Approved, though I did have one question which needs resolving.

**Pull request merged**